src/components/Form/CreateAccount.tsx

```diff
@@ -52,6 +58,8 @@ function validateFormValues(formValues: AccountCreationValues) {
   }
   if (!formValues.createNewKey && !formValues.privateKey.match(/^S[A-Z0-9]{55}$/)) {
     errors.privateKey = new Error("Invalid stellar private key.")
+  } else if (!formValues.createNewKey && isAccountAlreadyImported(formValues.privateKey, accounts)) {
+    errors.privateKey = new Error("You can not import the same account twice.")
```

Comment:
```suggestion
    errors.privateKey = new Error("This account is already set up.")
```

?

Reply:
"You cannot import the same account twice" is good. Or "This account is already in your wallet".

Reply:
"set up" sounds like an installation. Not appropriate.